redis/src/script.rs

```diff
@@ -213,7 +213,15 @@ impl<'a> ScriptInvocation<'a> {
     }
 
     fn eval_cmd(&self) -> Cmd {
-        let mut cmd = cmd("EVALSHA");
+        let args_len = 3 + self.keys.len() + self.args.len();
+        let buf_len = self
+            .keys
+            .iter()
+            .chain(self.args.iter())
+            .fold(0, |acc, e| acc + e.len())
+            + 7 /* "EVALSHA".len() */
+            + self.script.hash.len();
+        let mut cmd = Cmd::with_capacity(args_len, buf_len);
         cmd.arg(self.script.hash.as_bytes())
```

Comment:
now you didn't set the "EVALSHA" argument, and this will fail. 😸

Reply:
🧎

I'll fix that.